### PR TITLE
fix #59

### LINKED ISSUE #59: customFilePath is receiving an array instead of an string
`exec` here: https://github.com/ludohenin/gulp-inline-ng2-template/blob/master/parser.js#L278

returns an array, this is being passed into `customFilePath` but it expects an `string`

### PR DESCRIPTION
**Pull request by vkennke**
fixes #59

### CODE REVIEW

File: parser.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ var cssOptions = function () {
     type: 'css',
     prop_url: 'styleUrls',
     prop: 'styles',
-    pattern: /styleUrls\s*:\s*\[([^\]]*)\]/g
+    pattern: /styleUrls\s*:\s*\[([^\]]*)]/g
```
Comment (ludohenin) [inline parser.js]: Why did you unescape this bracket ?

**Comment by ludohenin** [comment]
I might wait a bit before versioning this one as this introduce a breaking change.

File: parser.js (hunk 1 above)
Comment (vkennke) [reply parser.js]: Ok, sorry, this line change wasn't indended to be pushed. Anyway, the closing square bracket should not be escaped in this regex: https://stackoverflow.com/a/43044993 (yeah, the question is about Java regex but it's also true for JS)

Comment (ludohenin) [reply parser.js]: ok, will merge soon. thanks

**Pull request merged**